src/target/llvm/codegen_llvm.cc

```diff
@@ -1127,6 +1128,13 @@ void CodeGenLLVM::SetTargetAttributes(llvm::Function* func) {
   if (!features.empty()) {
     func->addFnAttr("target-features", features);
   }
+#if TVM_LLVM_VERSION >= 130
+  // Add vscale_range() function attribute when appropriate.
+  if (llvm_target_->TargetHasCPUFeature("sve") || llvm_target_->TargetHasCPUFeature("sme")) {
```

Comment:
Apologies, I had another thought about this. Since it's AArch64 specific, perhaps we should move it to `codegen_aarch64.cc`?

Reply:
Done!